src/rendering/SilkyNvg.Rendering.Vulkan/Pipelines/PipelineSettings.cs

```diff
@@ -21,6 +22,70 @@ internal struct PipelineSettings
         public StencilOp BackStencilDepthFailOp;
         public StencilOp BackStencilPassOp;
 
+        public bool Equals(PipelineSettings other)
```

Comment:
Feels like this might make sense being a `record struct` so all this gets generated for us.